### PR TITLE
cycles = 0

### PR DESCRIPTION
**Pull request by alanlujan91**

### CODE REVIEW

**Comment by alanlujan91** [comment]
Using dask to speed up numerical exercise; parallel processing of function calls scaled up to computer resources.

**Comment by sbenthall** [comment]
cool.

let me know when this is ready to merge.

**Comment by alanlujan91** [comment]
@sbenthall i removed some of your code, especially the plotting, to make this faster. Is that ok? Do we need some sort of plot for this?

**Comment by sbenthall** [comment]
i think the plotting was just an artifact; no worries

File: simulate/parameters.py [hunk 1]
```diff
@@ -84,20 +84,29 @@ def build_population(agent_type, parameters, seed=None, dphm=1500):
...
-    "Rfree": 1.0,
-    ### These are placeholders that will be set when the system is set up.
-    "CRRA": 6,
-    "DiscFac": 0.90,
-    "ex_post": None,  # ex post heterogeneous parameters over which to merge solutions
-})
+lucas0_agent_population_params.update(
+    {
+        "cycles": 0,  # issue 186
+        "aNrmInitStd": 0.28901524,  # calculated using dashboard from default init_portfolio stats
```
Comment (sbenthall) [inline simulate/parameters.py:99]: Is this supposed to be the aNrmInitMean ?

File: simulate/parameters.py [hunk 2]
```diff
@@ -84,20 +84,29 @@ def build_population(agent_type, parameters, seed=None, dphm=1500):
...
+        # "aNrmInitMean": 6
+        # "LivPrb": [0.98**0.25], -- Should this be adjusted?
+        "PermGroFac": 1.0,
+        "Rfree": 1.0,
+        ### These are placeholders that will be set when the system is set up.
+        "CRRA": 5,
```
Comment (sbenthall) [inline simulate/parameters.py:105]: This value is 6 in the Numerical notebook. Which do you mean?

File: simulate/parameters.py (hunk 1 above)
Comment (alanlujan91) [reply simulate/parameters.py:99]: I didn't add this here, but I think it should be setting aNrmInitMean

Comment (sbenthall) [reply simulate/parameters.py:99]: I agree. Would you mind correcting that? I would do it myself but it's in your personal PR.

File: simulate/parameters.py (hunk 2 above)
Comment (alanlujan91) [reply simulate/parameters.py:105]: it is 5 in the default so let's go with 5

**Comment by sbenthall** [comment]
is this ready to merge?

**Comment by alanlujan91** [comment]
Need to actually calculate aNrmInitMean, which notebook does this?

**Comment by sbenthall** [comment]
I don't think we've settled on a process for this.
There is code that does something like this in the numerical buffer stock and dashboard notebooks.
But there's nothing that says specifically:
 - here's the parameters we are committed to
 - here's the calculations we are doing to flesh out those parameters.

certainly open to improvements.

**Comment by alanlujan91** [comment]
Ok ready to review

**Comment by sbenthall** [comment]
Just to double check:
... are these dividend growth and stdev rates correct? i.e., did you use the same ones when calculating the target wealth?

https://github.com/sbenthall/SHARKFin/blob/master/simulate/run_any_simulation.py#L109-L118

honestly it might be better to have all these details settled in one document.

the configuration is scattered across many different points but we need a canonical description of what we're trying to do. This info will also go into the paper.

**Comment by alanlujan91** [comment]
@sbenthall on scaling... we can't scale income.... mNrm_next = a * R + 1*p_shk, we would need mNrm_next = a * R_scaled + 1/4 * p_shk_scaled

**Comment by sbenthall** [comment]
Could we:
 - start with a pLvl scaled down?
 - scale the income shocks down appropriately?

Not sure I'm following the logic here, but maybe you can run the question by Chris?

**Comment by alanlujan91** [comment]
@sbenthall ready to review

see `macro/macro_parameters.py` for new master list of parameters, annual and quarterly

**Comment by sbenthall** [comment]
Ok, merging now, thanks.

**Pull request merged**